addon/server.js

```diff
@@ -153,6 +158,8 @@ export default class Server {
     this.timing = this.timing || config.timing || 400;
     this.namespace = this.namespace || config.namespace || '';
     this.urlPrefix = this.urlPrefix || config.urlPrefix || '';
+    this.persistence = !this.isTest() && config.addonConfig && config.addonConfig.persistence;
```

Comment:
Can we change the ENV option to `persistDb` (instead of `persistence`)?

```js
ENV['ember-cli-mirage'] = {
  enabled: true,
  persistDb: true
};
```

feel like it gives a bit more clarity.

We can still keep the `persistence` naming throughout the internal implementation here.